.jazzy.yaml

```diff
@@ -0,0 +1,72 @@
+module: Blueprint
```

Comment:
Is this file generated, or manually maintained? (not super familiar with how jazzy works)

Reply:
Manual unfortunately